### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by DavevonH**

### CODE REVIEW

**Review by B-Baumeister — APPROVED** [review]
LGTM! =)

**Comment by shrocket** [comment]
You have an `<h1>` element with some text in it, but no CSS styling (are we supposed to?). The title *an sich* looks good. Also, there is no `<nav>` section. Reading through the instructions again, it would seem there is supposed to be one, but maybe that it should be `<hidden>`?

**Review by shrocket — APPROVED** [review]
LGTM!

File: index.html [hunk 1]
```diff
@@ -8,7 +8,33 @@
...
+      <h1>Capital City Quiz</h1>
     </header>
+    <main>
+      <section>
+        <h2>What is the capital of the US?</h2>
+        <p>Washington DC</p>
+        <ul>
+          <li>Capitals</li>
+          <li>United States</li>
+        </ul>
```
Comment (shrocket) [inline index.html:20]:
```suggestion
        <ul>
          <li>Capitals</li>

        </ul>
```

File: index.html [hunk 2]
```diff
@@ -8,7 +8,33 @@
...
   <body>
     <header>
-      <h1>Quiz App</h1>
+      <h1>Capital City Quiz</h1>
     </header>
+    <main>
+      <section>
+        <h2>What is the capital of the US?</h2>
+        <p>Washington DC</p>
+        <ul>
```
Comment (shrocket) [inline index.html:17]: this is beautiful

File: index.html [hunk 3]
```diff
@@ -8,7 +8,33 @@
...
+        </ul>
+      </section>
+      <section>
+        <h2>What is the capital of Sweden?</h2>
+        <p>Stockholm</p>
+        <ul>
+          <li>Capitals</li>
+          <li>Europe</li>
+        </ul>
+      </section>
```
Comment (shrocket) [inline index.html:29]:
```suggestion
      <section>
        <h2>What is the capital of Sweden?</h2>
        <p>Stockholm</p>
        <ul>
          <li>Capitals</li>

        </ul>
      </section>
```

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,62 @@
+* {
+  box-sizing: border-box;
+  font-family: sans-serif;
+}
+body {
+  background-color: ;
```
Comment (shrocket) [inline styles.css:6]: I think we try to avoid empty commands, no?

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  background-color: rgb(166, 163, 163);
+  display: flex;
+  justify-content: center;
+  border-radius: 5px;
+}
+
+h1 {
+  color: rgb(255, 208, 0);
+  text-decoration: solid;
+  text-shadow: 1px 1px red;
```
Comment (shrocket) [inline styles.css:19]: said before, will say again: I like the colors of your drop-shadow!

**Review by shrocket — CHANGES_REQUESTED** [review]
Please have a look at my minimal commentary and decide for yourself how you'd like to proceed!

File: styles.css (hunk 4 above)
Comment (DavevonH) [reply styles.css:6]: Thank you! I forgot this one :D

**Review by FelineHuhn — APPROVED** [review]
This is a huge first PR, which is more a size of the PRs (1 for the title, 1 for the quizcard, 1 for the navigation), but well done 🚀 
In the future it would be great, if you work in smaller steps. After an approval you should merge and delete the branch and locally run git pull and than create a new branch to work on it.

**Comment by DavevonH** [comment]
> This is a huge first PR, which is more a size of the PRs (1 for the title, 1 for the quizcard, 1 for the navigation), but well done 🚀
> In the future it would be great, if you work in smaller steps. After an approval you should merge and delete the branch and locally run git pull and than create a new branch to work on it.

I know it was a bit to much 😬 I will try my best to do so from now on! Thanks for the feedback! 😎

**Pull request merged**